### ISSUE TITLE
[FR] New line between top-level tags

### ISSUE THREAD

**Issue by wenfangdu** [vuejs/eslint-plugin-vue#2094]
**Please describe what the rule should do:**

Enforce a new line between top-level tags, e.g. `<script>`, `<template>`, etc.

**What category should the rule belong to?**

[x] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```html
<!-- ✓ GOOD -->
<script>
  /* ... */
</script>

<template>...</template>

<style>
  /* ... */
</style>

<!-- ✗ BAD -->
<style>
  /* ... */
</style>
<script>
  /* ... */
</script>
<template>...</template>
```

**Additional context**

NA

**Comment by FloEdelmann** [comment]
Already exists: https://eslint.vuejs.org/rules/padding-line-between-blocks.html

**Comment by wenfangdu** [comment]
@FloEdelmann Excellent!

**Comment by FloEdelmann** [comment]
Feel free to send a PR to updating the documentation of that rule, to add the keywords/description you were searching for.
